include/bringauto/structures/DeviceIdentification.hpp

```diff
@@ -84,16 +96,43 @@ class DeviceIdentification {
 	 */
 	[[nodiscard]] InternalProtocol::Device convertToIPDevice() const;
 
+	/**
+	 * TODO
+	 * @return
+	 */
+	[[nodiscard]] std::string convertToString() const {
+		return module_ + "/" + std::to_string(deviceType_) + "/" + deviceRole_ + "/" + deviceName_;
+	}
+
 private:
 	/// Module number
-	uint32_t module_;
+	uint32_t module_ {};
 	/// Device type
-	uint32_t deviceType_;
+	uint32_t deviceType_ {};
 	/// Role of the device
-	std::string deviceRole_;
+	std::string deviceRole_ {};
 	/// Name of the device
-	std::string deviceName_;
+	std::string deviceName_ {};
 	/// Priority of the device
-	uint32_t priority_;
+	uint32_t priority_ {};
+};
+
+
+}
+
+template <>
```

Comment:
Should we let the hash spec. here or move it into a separate file?

Reply:
I would keep it in the same file